hugo/layouts/events/single.html

```diff
@@ -30,7 +30,7 @@ <h4 class="card-title">Contents</h4>
     <small><a href="#" class="text-muted"><i class="fas fa-arrow-up"></i> up</a></small>
     <h4 class="pb-2">
       <span class="mr-2">
-      {{- with $volume.url -}}
+      {{- if $volume.url -}}
```

Comment:
That looks wrong. What happened here?

Reply:
But that's exactly what the `with...` statement does. It is functionally the same as `if isset ...`, except that it sets the dot `.` to the variable content within the block as well.

In your current version, the code is simply wrong, as the `.` on the next line won't be set to `$volume.url` anymore.